webapp/src/Controller/API/GroupController.php

```diff
@@ -89,14 +87,23 @@ public function singleAction(Request $request, string $id): Response
         description: 'Returns the added group',
         content: new Model(type: TeamCategory::class)
     )]
-    public function addAction(Request $request, ImportExportService $importExport): Response
-    {
+    public function addAction(
+        #[MapRequestPayload(validationFailedStatusCode: Response::HTTP_BAD_REQUEST)]
```

Comment:
Do we have to add this everywhere?

Reply:
The default is a 422. If you can live with that we can drop the `validationFailedStatusCode` part, but then we need to update the tests (since we sometimes return a 400 and sometimes a 422) or we always need to return a 422.